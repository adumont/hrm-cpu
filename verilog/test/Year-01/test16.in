31 3b
03 d3
34 46
23 31
48 cb
29 f6
35 2d
37 d7
