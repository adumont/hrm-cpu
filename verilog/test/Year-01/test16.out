3b
d3
46
